src/components/Checkbox/Checkbox.js

```diff
@@ -118,35 +119,42 @@ const CheckboxLabel = styled('label')`
 /**
  * Checkbox component for forms.
  */
-const Checkbox = ({ onToggle, children, name, ...props }) => (
-  <Fragment>
-    <CheckboxInput
-      id={name}
-      onClick={onToggle}
-      type="checkbox"
-      {...{ ...props, name }}
-    />
-    <CheckboxLabel htmlFor={name} {...props}>
-      {children}
-    </CheckboxLabel>
-  </Fragment>
-);
+const Checkbox = ({ onToggle, children, id, ...props }) => {
+  const defaultId = uniqueId('checkbox_');
```

Comment:
We can't inline these, because `uniqueId()` returns a new unique id every time it is called. We need one id to connect the label and input though. That's why we need to do it outside of the JSX.